metrics/coverage_converter/docs/02_coverage_converter_publishing.md

```diff
@@ -0,0 +1,230 @@
+- Download using the command line commands.
+
+Let's consider each of them separately: 
+
+## Downloading using the GitHub GUI
+
+To download the Coverage Converter tool from the GitHub releases, follow the next steps: 
+
+1. Go to the platform-platform/monorepo GitHub repository [releases page](https://github.com/platform-platform/monorepo/releases). 
+2. Find the version of the Coverage Converter tool you want to download.
+3. Open the `Assets` collapsible section and choose the Coverage Converter file compatible with your OS.
+4. Click to download
+
+## Downloading using the command line commands
+
+If you want to download the Coverage Converter tool from the CI, for examples, or just using the command line, you should follow the steps below: 
+
+1. Open the GitHub repository releases and find the release you want to download, as described in the [Downloading using the GitHub GUI](#Downloading-using-the-GitHub-GUI) section.
+2. Find the executable file compatible with your OS
+3. Right-click on this file and copy the link address. 
+
+After these steps, you should obtain the download link similar to this one: 
+
+`https://github.com/platform-platform/monorepo/releases/download/v1.0.0/coverage_converter_macos`
+
+As you can see, the link contains the release version. You can modify the link to make it refer to the latest release by removing the version from the URL and adding the `latest` keyword after the `releases`. So, the link should look like this: 
+
+`https://github.com/platform-platform/monorepo/releases/latest/download/coverage_converter_macos`
+
+## Using the Coverage Converter tool
+
+Once you've obtained the downloading link, you can download the Coverage Converter tool using the command line commands. Let's consider the macOS command-line command to download the Coverage Converter tool: 
+
+`curl -o <output> -k <url>`
+
+Where the `<output>` is the file path where you want to download the coverage converter, and the `<url>` is the download URL obtained previously. 
+
+So, when you've got the Coverage Converter executable, you probably, want to convert coverage to be readable by the CI integrations tool. To run the conversion process, you should have the coverage report in one of the supported formats: 
+
+- `LCOV`
+- `Istanbul`
+
+Let's consider the command for converting the `LCOV` format for macOS: 
+
+`./coverage_converter_macos lcov -i <YOUR_COVERAGE_REPORT_FILE> -o <COVERAGE_REPORT_OUTPUT_FILE.json>`
+
+As you can see, the Coverage Converter CLI contains a separate command for each supported coverage format. So, to convert the `Istanbul` coverage report, you should replace the `lcov` command with `istanbul` and so on. Also, let's review the CLI arguments: 
+
+- `-i, --input` - the parameter to specify the input coverage file path. This parameter is required, and the CLI cannot be run without this argument.
+- `-o, --output` - the parameter to specify the output coverage file path. This parameter could be 
```

Comment:
```suggestion
- `-o, --output` - the parameter to specify the output coverage file path. This parameter could be omitted.
```

Maybe we should link to the Coverage Converter tool design here so we don't duplicate parameter description?